File: vericred/backend/app/api/routes/ipfs.py
from fastapi import APIRouter

router = APIRouter(
    prefix="/api/ipfs",
    tags=["IPFS"],
)


@router.get("/{cid}")
async def get_ipfs_metadata(cid: str):
    return {
        "cid": cid,
        "message": "IPFS metadata endpoint ready",
    }

File: vericred/frontend/src/utils/ipfs.js
export function ipfsToHttp(cid){ if(!cid) return ""; if(cid.startsWith("http")) return cid; if(cid.startsWith("ipfs://")) cid=cid.slice(7); return `https://ipfs.io/ipfs/${cid}`; }
export default ipfsToHttp;
